src/utils/fetchClients.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
don't leave comments or disable eslint

Reply:
if I remove this commend its warning about Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

Reply:
so fix the error)

Reply:
so fix the error)